chronicles.nimble

```diff
@@ -8,8 +8,12 @@ license       = "Apache License 2.0"
 skipDirs      = @["tests"]
 
 requires "nim >= 1.6.0"
-requires "testutils"
 requires "json_serialization"
 
+when NimMajor >= 2:
+  taskRequires "test", "testutils"
```

Comment:
What does the `"test"` do here? It doesn't seem to refer to a package from https://github.com/nim-lang/packages/blob/master/packages.json

@jangko

Reply:
Ahh, got it, the first argument is the task that the dependency is being attached to. So, the `"test"` is referring to the `task test` down below, not to a package!